gallery/plot_visualization_utils.py

```diff
@@ -99,19 +99,19 @@ def show(imgs):
 # Visualizing segmentation masks
 # ------------------------------
 # The :func:`~torchvision.utils.draw_segmentation_masks` function can be used to
-# draw segmentation amasks on images. We can set the colors as well as
-# transparency of masks.
-#
-# Here is demo with torchvision's FCN Resnet-50, loaded with
-# :func:`~torchvision.models.segmentation.fcn_resnet50`.
-# You can also try using
-# DeepLabv3 (:func:`~torchvision.models.segmentation.deeplabv3_resnet50`)
-# or lraspp mobilenet models
+# draw segmentation amasks on images.
+# 
+# Semantic segmentation models
```

Comment:
@datumbox To illustrate why natively handling floats images [0, 1] is useful see below how we never use the uint8 images. We only use the batch (floats in 0, 1) and the normalized batch.

The only reason we still need the uint8 images in this example is because `draw_bounding_boxes` doesn't support floats yet, but I intend to update it as well. Once done, we will be able to completely remove references to the  `uint8` images.

Reply:
If this PR requires the other one, consider merging the two or stacking them. Given the discussion on the other thread, it's worth showing end-to-end your proposal and that might lead to faster merging.

Reply:
This PR in already a superset of #3820 the changes of #3820 are included here, but I think they should be merged separately